nomad/node_endpoint.go

```diff
@@ -944,7 +944,7 @@ func (n *Node) DeriveVaultToken(args *structs.DeriveVaultTokenRequest,
 	// setErr is a helper for setting the recoverable error on the reply and
 	// logging it
 	setErr := func(e error, recoverable bool) {
-		reply.Error = structs.NewRecoverableError(e, recoverable).(*structs.RecoverableError)
+		reply.Error = structs.NewRecoverableError(e, recoverable)
```

Comment:
I was going to do a type assertion before I thought I could just change the return type.
Will have shortly.